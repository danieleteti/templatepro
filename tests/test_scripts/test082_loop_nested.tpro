************* Nested List of objects
{{for dataitem in dataitems}}
1° Level |{{:dataitem.PropStr}}|{{:dataitem.PropInt}}|
    {{for dataitem2 in dataitem.dataitemlist}}
        2° Level |{{:dataitem2.Value1}}|
    {{endfor}}
---
{{endfor}}


** SIMPLE NESTED **
{{:nested | json}}
{{exit}}
{{:nested.ValueNested1}}
{{:nested.SimpleNested2.ValueNested2}}