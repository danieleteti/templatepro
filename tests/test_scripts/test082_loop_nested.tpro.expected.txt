************* Nested List of objects
1° Level |Str0|0|
            2° Level |SimpleDataItem0|
            2° Level |SimpleDataItem1|
            2° Level |SimpleDataItem2|
    ---
1° Level |Str1|1|
            2° Level |SimpleDataItem0|
            2° Level |SimpleDataItem1|
            2° Level |SimpleDataItem2|
    ---
1° Level |Str2|2|
            2° Level |SimpleDataItem0|
            2° Level |SimpleDataItem1|
            2° Level |SimpleDataItem2|
    ---


** SIMPLE NESTED **
ValueNested.1
ValueNested.1.2
ValueNested.1.2.3